index.html

```diff
@@ -4,33 +4,186 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <meta name="description" content="Struggling to find a gift for your wife or girlfriend? No problem!">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
-    <link rel="stylesheet" href="./css/main.css">
+    <link rel="stylesheet" href="/style/main.css">
+    <link rel="stylesheet" href="/fonts/style.css">
   </head>
   <body>
     <header class="page-header">
       <div class="container header-flex">
         <a href="#" class="page-logo">
-          <img src="1222" alt="wedding logo">
+          <img src="/images/logo.png" alt="wedding logo">
         </a>
         <nav class="main-nav">
           <ul class="main-nav__list">
             <li class="main-nav__list-item">
-              <a href="#" class="main-nav__link">Home</a>
+              <a href="#1" class="main-nav__link">Home</a>
             </li>
             <li class="main-nav__list-item">
-              <a href="#" class="main-nav__link">About Us</a>
+              <a href="#2" class="main-nav__link">About Us</a>
             </li>
             <li class="main-nav__list-item">
-              <a href="#" class="main-nav__link">Services</a>
+              <a href="#3" class="main-nav__link">Services</a>
             </li>
             <li class="main-nav__list-item">
-              <a href="#" class="main-nav__link">Blogs</a>
+              <a href="#4" class="main-nav__link">Blogs</a>
             </li>
           </ul>
         </nav>
-        <button class="main-button">gift finder</button>
+        <button class="header-button">gift finder</button>
       </div>
     </header>
+    <main>
+      <section class="hero-section">
+        <div class="container hero-flex">
+          <div class="hero-col col-element">
+            <h1 id="1" class="main-title">Find a gift for your wife or girlfriend</h1>
```

Comment:
Will you recall what id='1' means after several weeks? Maybe use id='heading_1''